packages/babel-plugin-makepot/package.json

```diff
@@ -0,0 +1,32 @@
+{
+	"name": "@wordpress/babel-plugin-makepot",
+	"version": "1.0.0-beta.0",
+	"description": "WordPress Babel i18n Plugin",
+	"author": "WordPress",
+	"license": "GPL-2.0-or-later",
+	"keywords": [
+		"i18n",
+		"babel",
+		"plugin"
+	],
+	"homepage": "https://github.com/WordPress/packages/tree/master/packages/babel-plugin-makepot/README.md",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/WordPress/packages.git"
+	},
+	"bugs": {
+		"url": "https://github.com/WordPress/packages/issues"
+	},
+	"main": "src/index.js",
```

Comment:
The root build configuration is such that a build distributable will be created for this, even if the `src/index.js` doesn't require transpilation. We should consider one of:

- Target `build/index.js` as `main`, even if not strictly necessary, merely for consistency with other packages
- Introduce a pattern for excluding particular projects from the Babel build step

Reply:
I'm using the build folders right now, let's add this option separately because it seems useful for other packages as well.